# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: target/mnemosyne/pkg_diff/runit-logacct_embutils.diff
# Copyright (C) 2002 - 2006 Alejandro Mery
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-06-20 17:26:41.000000000 +0200
+++ ./Makefile	2006-06-20 17:46:34.000000000 +0200
@@ -9,7 +9,7 @@
 	@echo "nothing to do yet"
 install:
 	install -m 755 src/sulogin.sh $(DESTDIR)$(SBINDIR)/sulogin
-	for f in last wall mesg; do \
+	for f in last wall; do \
 		install -m 755 src/$$f.sh $(DESTDIR)$(BINDIR)/$$f ; \
 	done
 	ln -snf last $(DESTDIR)$(BINDIR)/lastb
